src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -213,15 +200,9 @@ export class DotEditContentHtmlService {
 
         dotEditContentletEl.innerHTML = `
             <div class="dotedit-contentlet__toolbar">
-                <button type="button" data-dot-identifier="${contentlet.identifier}" data-dot-inode="${
-            contentlet.inode
-        }" class="dotedit-contentlet__edit">Edit</button>
-                <button type="button" data-dot-identifier="${contentlet.identifier}" data-dot-inode="${
-            contentlet.inode
-        }" class="dotedit-contentlet__remove">Remove</button>
-                <button type="button" data-dot-identifier="${contentlet.identifier}" data-dot-inode="${
-            contentlet.inode
-        }" class="dotedit-contentlet__drag">Drag</button>
+                <button type="button" data-dot-identifier="${contentlet.identifier}" data-dot-inode="${contentlet.inode}" class="dotedit-contentlet__drag">Drag</button>
```

Comment:
Is this 120 cols?

Reply:
maybe this should go for a new method some like 'createEditContentletToolbar' o maybe even other service